packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -31,6 +32,11 @@ export namespace VscodeCommands {
         label: 'VSCode open link'
```

Comment:
Also this one, should a user see it in the command palette?

Reply:
Not sure, will clarify about this one.

Reply:
PR provided in above comment.